README.md

```diff
@@ -106,6 +106,23 @@ end
 
 Operator match data includes `match['operator']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` setting.
 
+### Threaded Messages
+
+You can reply to a message in a thread (or keep sending to the current thread) :
+
+```ruby
+command 'reply in thread' do |client, data, match|
+  client.say(
+    channel: data.channel,
+    text: "let's avoid spamming everyone, I will tell you what you need in this thread",
+    thread_ts: data.thread_ts || data.ts
```

Comment:
Add a comment to what these are? Maybe a link to docs?

Reply:
Hello sorry for the long wait. I'm a French guy so sometimes I forget there is no space before semicolons in english ^^"

I have included extra information and links regarding `thread_ts` and `ts`.

Sorry for stalling this PR for so long